bcgov/biohubbc-platform
# SIMSBIOHUB-426: Calculate Submission Regions

## Links to Jira Tickets

[SIMSBIOHUB-426](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-426)

## Description of Changes

- Added migrations for `region_lookup` and `submission_regions`
- Added seed files for existing regions (taken from SIMS)
- Modified Submission fetch to include regions 
- Modified intake to include region calculations

## Testing Notes

- Run both SIMS and BioHub locally
- Create a Project/ Survey in SIMS 
- Publish that Survey
- In BioHub navigate to Submissions tab and see regions on new submission
